sacrebleu.py

```diff
@@ -1592,30 +1597,38 @@ def main():
     if args.langpair is not None and args.langpair.split('-')[1] == 'zh' and 'bleu' in args.metrics and args.tokenize != 'zh':
         logging.warning('You should also pass "--tok zh" when scoring Chinese...')
 
-    if args.test_set:
-        _, *ref_files = download_test_set(args.test_set, args.langpair)
-        if len(ref_files) == 0:
-            print('No references found for test set {}/{}.'.format(args.test_set, args.langpair))
-            sys.exit(1)
+    # concat_ref_files is a list of list of reference filenames, for example:
+    # concat_ref_files = [[testset1_refA, testset1_refB], [testset2_refA, testset2_refB]]
+    # Note that concatenation is possible only for internal test sets,
+    # which do not currently support multiple references, so the example is hypothetical.
+    if args.test_set is None:
+        concat_ref_files = [args.refs]
     else:
```

Comment:
This isn't right—we do have multiple references for some test sets, see `wmt17/tworefs`.

Reply:
Is there anything to say here? I think this is the only blocker on this one.

Reply:
You are right, I checked that `-t wmt16/tworefs,wmt17/tworefs -l en-fi` works as expected and deleted the misleading comment.
 (I had thought that multiple refs can be only specified as multiple files on the command line or a single file with tabs, but now I see that it is supported also for the internal test sets.)